ompi/mca/mtl/psm2/mtl_psm2_component.c

```diff
@@ -361,6 +365,8 @@ ompi_mtl_psm2_component_init(bool enable_progress_threads,
     int verno_minor = PSM2_VERNO_MINOR;
     int local_rank = -1, num_local_procs = 0;
 
+    OBJ_CONSTRUCT(&mtl_psm2_mq_mutex, opal_mutex_t);
```

Comment:
you did the static init above, so I don't believe you need to construct it again here

Reply:
Ok, will remove and re-test. OBJ_DESTRUCT will also fly away.